indexer/src/main/scala/com/expedia/www/haystack/trace/indexer/writers/es/ServiceMetadataWriter.scala

```diff
@@ -32,11 +33,26 @@ import io.searchbox.client.{JestClient, JestClientFactory}
 import io.searchbox.core.{Bulk, Index}
 import io.searchbox.indices.template.PutTemplate
 import io.searchbox.params.Parameters
+import org.apache.commons.lang3.time.FastDateFormat
 import org.slf4j.LoggerFactory
 
 import scala.collection.JavaConverters._
 import scala.util.Try
 
+object ServiceMetadataUtils {
+
+  private val timezone = TimeZone.getTimeZone("UTC")
+  private val format = FastDateFormat.getInstance("yyyy-MM-dd", timezone, Locale.US)
+
+  // creates an index name based on current date. following example illustrates the naming convention of
+  // elastic search indices for service metadata:
+  // service-metadata-2019-02-20
+  def indexName(prefix: String): String = {
+    val currentTime = Calendar.getInstance(timezone)
```

Comment:
same comment to get rid of Calendar.getInstance(). Use joda datetime

Reply:
done